src/Composer/Installers/QuicksilverInstaller.php

```diff
@@ -0,0 +1,9 @@
+<?php
+namespace Composer\Installers;
+
+class QuicksilverInstaller extends BaseInstaller
```

Comment:
It may be worth it to add the prefix to the name to uniquely identify the identity of the installer to Pantheon? For example: `PantheonQuicksilverInstaller`.